lib/checks/forms/autocomplete-valid.json

```diff
@@ -4,8 +4,9 @@
   "metadata": {
     "impact": "serious",
     "messages": {
-      "pass": "The autocomplete attribute is correctly formatted",
-      "fail": "The autocomplete attribute is incorrectly formatted"
+      "pass": "the autocomplete attribute is correctly formatted",
+      "fail": "the autocomplete attribute is incorrectly formatted",
+      "incomplete": "the autocomplete attribute has a non-standard value. Check whether any standard value could be used instead."
```

Comment:
I struggled to understand what the meaning of this message was from the recent suggestion. I hope this clarifies it but we can discuss if you want this changed @WilcoFiers